content/en/docs/concepts/policy/pod-security-policy.md

```diff
@@ -22,7 +22,7 @@ updates.
 ## What is a Pod Security Policy?
 
 A _Pod Security Policy_ is a cluster-level resource that controls security
-sensitive aspects of the pod specification. The `PodSecurityPolicy` objects
+sensitive aspects of the pod specification. The [PodSecurityPolicy](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podsecuritypolicy-v1beta1-policy) objects
```

Comment:
Rather than this link, I recommend adding an API link to a new _What's next_ section at the foot of the page.